src/workerd/io/compatibility-date.capnp

```diff
@@ -337,4 +337,8 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatEnableFlag("rtti_api")
       $experimental;
   # Enables the `workerd:rtti` module for querying runtime-type-information from JavaScript.
+
+  jsRpcStub @35 :Bool
+      $compatEnableFlag("durable_object_js_rpc");
```

Comment:
You need to mark this `$experimental` to make sure people cannot deploy code using it.

Alternatively, as long as you are not making breaking changes, you don't really have to declare a new compat flag, you can reuse the `workerdExperimental` flag.

IIRC we decided that when this is actually released, stubs don't need a compat flag, only the server side does. In that case you should reuse `workerdExperimental` for the stub side.